frontend/apps/web/src/pages/system/_name.tsx

```diff
@@ -28,95 +33,224 @@ const RepositoryDetails = () => {
+                      onClick={() => null}
+                    />
+                  )
+                })}
+              </ColumnLayout>
+            )
+          })}
+
+          <Legend>
+            <div>
+              Missing tests <div />
+            </div>
+            <div>
+              Passing tests <div />
+            </div>
+            <div>
+              Failing tests <div />
+            </div>
+          </Legend>
+        </StackLayout>
+      </BoxLayout>
+    </DetailedView>
+  )
+
   return (
     <PageLayout {...pageLayout}>
       <PageLayout.Section>
         <Text type={'subtitle l'} color={'gray80'}>
           {totalTests} {t('filecoin.system.totalTests')}
         </Text>
-        <DetailedView>
-          <ReactTooltip
-            effect="solid"
-            getContent={data => {
-              const { functionName, path, repository } = JSON.parse(data) || {}
-              return (
-                <>
-                  <div>
-                    <b>Name</b>: <span>{functionName}</span>
-                  </div>
-
-                  <div>
-                    <b>Path</b>: <span>{path}</span>
-                  </div>
-
-                  <div>
-                    <b>Repository</b>: <span>{repository}</span>
-                  </div>
-                </>
-              )
-            }}
-          />
-
-          <BoxLayout gap={1.5}>
-            <StackLayout gap={1}>
-              <ColumnLayout className={'c-matrix__header'} gap={1}>
-                {testKinds.map(testKind => {
-                  return <Text key={testKind}>{testKind}</Text>
-                })}
-              </ColumnLayout>
-
-              {system.subsystems.map(subsystem => {
-                return (
-                  <ColumnLayout
-                    className={'c-matrix__row'}
-                    gap={1}
-                    key={subsystem.name}
-                  >
-                    <Text>{subsystem.name}</Text>
-
-                    {testKinds.map(testKind => {
-                      const tests = subsystem.tests.filter(
-                        test => test.kind === testKind,
-                      )
-
-                      return (
-                        <MatrixMap
-                          key={testKind}
-                          data={tests}
-                          onClick={() => null}
-                        />
-                      )
-                    })}
-                  </ColumnLayout>
-                )
-              })}
-
-              <Legend>
-                <div>
-                  Missing tests <div />
-                </div>
-                <div>
-                  Passing tests <div />
-                </div>
-                <div>
-                  Failing tests <div />
-                </div>
-              </Legend>
-            </StackLayout>
-          </BoxLayout>
-        </DetailedView>
+        {activeTab === 1 ? renderOverview() : renderDetailedView()}
```

Comment:
Create a component `<DetailedView {...here add props needed (system, testKinds, etc...}  />` and `<Overview/>`
You can add component in path:
`frontend/apps/web/src/components/system/DetailedView.tsx`

Reply:
Done